.github/scripts/closeNeedMoreInfoIssues.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+const {
+  labels: { planning },
+} = require("./support/resources");
+
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+module.exports = async ({ github, context }) => {
+  const { repo, owner } = context.repo;
+  const DAYS_BEFORE_CLOSE = 14;
+
+  console.log(`Checking for issues with the label: "${planning.needsInfo}" that are stale.`);
+
+  const { data: issues } = await github.rest.issues.listForRepo({
+    owner: owner,
+    repo: repo,
+    state: "open",
+    labels: planning.needsInfo,
+    per_page: 100,
+  });
+
+  const now = new Date();
+
+  for (const issue of issues) {
+    const lastUpdated = new Date(issue.updated_at);
+    const daysSinceUpdate = Math.round((now.getTime() - lastUpdated.getTime()) / (1000 * 60 * 60 * 24));
```

Comment:
Suggestion: Adding a var for days in milliseconds could help with readability. It looks readable to me, so I'll leave it up to you.